taxi/models.py

```diff
@@ -0,0 +1,31 @@
+from django.conf import settings
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+
+    class Meta:
+        verbose_name = "Driver"
+        verbose_name_plural = "Drivers"
```

Comment:
did u read checklist ?

Reply:
Yes but I don't understand what I should do )

Reply:
You have noticed that Driver inherits from AbstractUser. It means that the standard User model should be replaced with the model Driver. Notice, that Driver has an additional field compared to the standard User. To make correct model name display in the admin, add verbose_name and verbose_name_plural inside Driver model.

Reply:
Sorry)